vmas/make_env.py

```diff
@@ -16,14 +16,16 @@ def make_env(
     device: DEVICE_TYPING = "cpu",
     continuous_actions: bool = True,
     wrapper: Optional[
-        Wrapper
-    ] = None,  # One of: None, vmas.Wrapper.RLLIB, and vmas.Wrapper.GYM
+        Union[Wrapper, str]
+    ] = None,  # One of: None, vmas.Wrapper.RLLIB, vmas.Wrapper.GYM, vmas.Wrapper.Gymnasium
     max_steps: Optional[int] = None,
     seed: Optional[int] = None,
     dict_spaces: bool = False,
     multidiscrete_actions: bool = False,
     clamp_actions: bool = False,
     grad_enabled: bool = False,
+    terminated_truncated: bool = False,
+    wrapper_kwargs: dict = {},
```

Comment:
```suggestion
    wrapper_kwargs: Optional[Dict] = None,
```

let's not use mutables as defaults as in python they casue a lot of trouble